src/Chat/Plugin/CommandOnlyPlugin.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Room11\Jeeves\Chat\Plugin;
+
+use Room11\Jeeves\Chat\Command\Message;
+
+/**
+ * Provides default interface method implementations for plugins which only handle commands
+ */
+trait CommandOnlyPlugin
+{
+    public function handleMessage(Message $message): \Generator { yield; }
```

Comment:
Good to know, although in this case I'm not sure if it's worthwhile because the point of this trait is that the method would never be called. I'd suggest this is OK because more readable (`if (false)` always makes me wtf) but I'm not married to it